src/manager.rs

```diff
@@ -68,10 +68,15 @@ impl<T: MutEventSubscriber> SubscriberOps for EventManager<T> {
 impl<S: MutEventSubscriber> EventManager<S> {
     /// Create a new `EventManger` object.
     pub fn new() -> Result<Self> {
+        Self::new_with_capacity(256)
+    }
+
+    /// Creates a new `EventManger` object with specified event capacity.
+    pub fn new_with_capacity(event_capacity: usize) -> Result<Self> {
```

Comment:
Can you rename `event_capacity` to `ready_events_capacity`?

Reply:
done 👍